docs/git-sync-relay.md

```diff
@@ -36,18 +36,18 @@ Create a `values.yaml` file with contents similar to the following:
 gitSyncRelay:
   enabled: True
   repo:
-    url: https://github.com/astronomer/airflow-example-dags
+    url: ssh+git://git@github.com/astronomer/2-4-example-dags.git # this can be https:// for public repositories
     branch: main
     depth: 1 # default to a shallow clone because it is faster, though it sacrifices git history
     wait: 60 # seconds between synchronizations with upstream git repo
     subPath: dags # if your dags dir is not the repo root, specify the path relative to the repo root
-    sshPrivateKeySecretName: theGitSshPrivateKeyName # The name of a secret that holds the private key.
+    sshPrivateKeySecretName: git-ssh-private-key # This is the secret we created earlier. This is not requred with https git remotes.
 ```
 
 ### Install airflow
 
-Then install airflow like you normally would. Assuming your cwd is the root dir or the astronomer/airflow-chart repo:
+You must also follow the git-sync configuration setup that you would normally follow to install the apache airflow chart. The one customization you must make is to use the git repo served by the git-sync-relay's git-daemon pod as your git remote. That URL is `git://airflow-git-sync-relay.${NAMESPACE}.svc.cluster.local./git` where `${NAMESPACE}` is the airflow installation namespace. Once you have those values set up in your values.yaml file, you can install the astronomer airflow-chart:
```

Comment:
An example might be helpful. Is there any auth on the relay?

Reply:
git-daemon serves using git protocol, which has no auth and is quite performant. security is maintained by keeping the git-daemon restricted to the local namespace.

What kind of example are you thinking of?

Reply:
Cool. I meant an example values, something like this maybe:

```
airflow:
  dags:
    gitSync:
      enable: true
      repo: git://airflow-git-sync-relay/git`
```

And to make it simpler, just using the service name should be okay right?

```suggestion
You must also follow the git-sync configuration setup that you would normally follow to install the apache airflow chart. The one customization you must make is to use the git repo served by the git-sync-relay's git-daemon pod as your git remote. That URL is `git://airflow-git-sync-relay/git`. Once you have those values set up in your values.yaml file, you can install the astronomer airflow-chart:
```

Reply:
@jedcunningham can you check the current version and see if it's suitable?

Reply:
Sorry, missed this! Looks good.